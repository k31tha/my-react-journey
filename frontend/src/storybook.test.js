import initStoryshots from '@storybook/addon-storyshots';
import "@testing-library/jest-dom/extend-expect";

initStoryshots();
