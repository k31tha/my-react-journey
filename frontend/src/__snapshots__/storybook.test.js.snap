// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ClubDetail Active Club 1`] = `
<div>
  <h1>
    Woking FC
  </h1>
  <span>
    <img
      alt="Woking FC"
      src="/assets/logos/WokingLogo.png"
    />
  </span>
  <br />
  <span>
    official website:
     
    <a
      href="https://wokingfc.co.uk/"
      target="_blank"
    >
      https://wokingfc.co.uk/
    </a>
  </span>
</div>
`;

exports[`Storyshots ClubDetail Active Club With No Website 1`] = `
<div>
  <h1>
    Woking FC
  </h1>
  <br />
  <span>
    official website: not known
  </span>
</div>
`;

exports[`Storyshots ClubFilterByStatus Default 1`] = `
Array [
  <button
    onClick={[Function]}
  >
    Active
  </button>,
  <button
    onClick={[Function]}
  >
    InActive
  </button>,
  <button
    onClick={[Function]}
  >
    All
  </button>,
]
`;

exports[`Storyshots ClubLink Active Club 1`] = `
<li
  className=""
>
  <a
    href="woking-fc"
  >
    Woking FC
  </a>
</li>
`;

exports[`Storyshots ClubLink In Active Club 1`] = `
<li
  className="InActiveClub"
>
  <a
    href="leyton-orient-fc"
  >
    Leyton Orient FC
  </a>
</li>
`;

exports[`Storyshots ClubLinkList Default 1`] = `
<ul>
  <li
    className=""
  >
    <a
      href="woking-fc"
    >
      Woking FC
    </a>
  </li>
  <li
    className="InActiveClub"
  >
    <a
      href="leyton-orient-fc"
    >
      Leyton Orient FC
    </a>
  </li>
</ul>
`;

exports[`Storyshots ClubLinkList Empty 1`] = `
<div
  className="list-items"
>
  empty list
</div>
`;

exports[`Storyshots ClubSearch Default 1`] = `
Array [
  "Search by:",
  <input
    aria-label="club-search"
    onChange={[Function]}
    type="text"
    value=""
  />,
  <br />,
  <br />,
  <button
    onClick={[Function]}
  >
    Active
  </button>,
  <button
    onClick={[Function]}
  >
    InActive
  </button>,
  <button
    onClick={[Function]}
  >
    All
  </button>,
  <br />,
  <ul>
    <li
      className=""
    >
      <a
        href="woking-fc"
      >
        Woking FC
      </a>
    </li>
    <li
      className="InActiveClub"
    >
      <a
        href="leyton-orient-fc"
      >
        Leyton Orient FC
      </a>
    </li>
    <li
      className=""
    >
      <a
        href="knaphill-fc"
      >
        Knaphill FC
      </a>
    </li>
  </ul>,
]
`;

exports[`Storyshots ClubSearch Empty 1`] = `
Array [
  "Search by:",
  <input
    aria-label="club-search"
    onChange={[Function]}
    type="text"
    value=""
  />,
  <br />,
  <br />,
  <button
    onClick={[Function]}
  >
    Active
  </button>,
  <button
    onClick={[Function]}
  >
    InActive
  </button>,
  <button
    onClick={[Function]}
  >
    All
  </button>,
  <br />,
  <div
    className="list-items"
  >
    empty list
  </div>,
]
`;

exports[`Storyshots ClubSearchByName Default 1`] = `
Array [
  "Search:",
  <input
    aria-label="club-search"
    onChange={[Function]}
    type="text"
    value="Knaphill"
  />,
]
`;
